### ISSUE TITLE
Handle EADDRINUSE error

### ISSUE THREAD

**Issue by adamdbradley** [driftyco/ionic-cli#250]
ionic run --livereload

If you try creating another server with the same address you get this: 

```
Running dev server: http://10.0.1.33:8100
 ✗
 ✗ ... Uhoh. Got error %s ... listen EADDRINUSE
 ✗ Error: listen EADDRINUSE
    at errnoException (net.js:901:11)
    at Server._listen2 (net.js:1039:14)
    at listen (net.js:1061:10)
    at Server.listen (net.js:1127:5)
    at Server.listen (/usr/local/lib/node_modules/ionic/node_modules/tiny-lr-fork/lib/server.js:138:15)
    at Object.IonicTask.start (/usr/local/lib/node_modules/ionic/lib/ionic/serve.js:178:16)
    at /usr/local/lib/node_modules/ionic/lib/ionic/cordova.js:160:15
    at Object.IonicTask.getAddress (/usr/local/lib/node_modules/ionic/lib/ionic/serve.js:683:9)
    at Object.IonicTask.loadSettings (/usr/local/lib/node_modules/ionic/lib/ionic/serve.js:80:8)
    at Object.IonicTask.setupLiveReload (/usr/local/lib/node_modules/ionic/lib/ionic/cordova.js:157:13)
    at Object.IonicTask.run (/usr/local/lib/node_modules/ionic/lib/ionic/cordova.js:23:14)
    at Object.Ionic.run (/usr/local/lib/node_modules/ionic/lib/ionic.js:549:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/ionic/bin/ionic:10:7)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:901:3
```

Rather have it catch this error and cleanly explain the problem.

**Comment by jbavari** [comment]
Upon inspection, this error spawns up from the tiny-lr-fork module - in which it will log its error but not throw anything up for the caller to report, [see the code here](https://github.com/mklabs/tiny-lr/blob/master/lib/server.js#L170)

**Comment by jbavari** [comment]
Closing, no action to take, due to no exception being thrown.

**Comment by jbavari** [comment]
This has been fixed in 1.3.13.
